k8s/bases/cse-import-ec-d2cc/cse-import-ec-d2cc-runner-deployment.yaml

```diff
@@ -1,7 +1,7 @@
 kind: Deployment
 apiVersion: apps/v1
 metadata:
-  name: cse-import-ec-d2cc-gridcapa-runner
+  name: cse-import-ec-d2cc-runner
```

Comment:
The name should also be updated in patches in file `k8s/overlays/azure/dev-iso-coreso-prod/kustomization.yaml`